src/test/java/ShopperCartAnnotationWithExtensionTest.java

```diff
@@ -0,0 +1,28 @@
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class ShopperCartAnnotationWithExtensionTest {
+    @Mock
+    private Cart mockedCart;
+
+    private Shopper underTest;
+
+    @BeforeEach
+    void init() {
```

Comment:
you can make method / class name self-descriptive ->

instead of 'init' -> "mockitoExtentionWillTakeCareOfMocksInitialization" :)